www/pages/working-with-organisations.md

```diff
@@ -13,62 +13,40 @@ Another advantage of organisations is that other users
 can be invited and they will get access to organisation's projects - it is not necessary to grant or revoke permissions of a single user
 manually for each project.
 
-### Creating an Organisation
+## Creating an Organisation
 
 This section describes how to create an organisation.
 
-1. Log in to Mergin
-2. Click the account settings button ![](../images/organisations/account-settings-button.png) in the top-right then click **Your Organisations**
+- Log in to Mergin
+- From the left panel, select **+ Create** in front of **Organisations**
+- Type a name for your **Organisation name**
+- Fill in the **Description**
+- Press **Save**
 
-	<p align="center"><img src="../images/organisations/organisations.png" width="90%"></p>
+<p align="center"><img src="../images/organisations/create-organisation.png"></p>
 
-3. Click the + button in the top right to add a new organisation
+**Note**: to be able to use organisation features, you need [subscribe to the Team plan](subscriptions) first.

```

Comment:
Maybe worth adding an explanation that it is because organisations have zero storage quota when created?